recipes/libatomic_ops/all/conandata.yml

```diff
@@ -1,10 +1,7 @@
 sources:
-  "7.6.10":
-    url: "https://github.com/ivmai/libatomic_ops/releases/download/v7.6.10/libatomic_ops-7.6.10.tar.gz"
-    sha256: "587edf60817f56daf1e1ab38a4b3c729b8e846ff67b4f62a6157183708f099af"
-  "7.6.12":
-    url: "https://github.com/ivmai/libatomic_ops/releases/download/v7.6.12/libatomic_ops-7.6.12.tar.gz"
-    sha256: "f0ab566e25fce08b560e1feab6a3db01db4a38e5bc687804334ef3920c549f3e"
   "7.6.14":
     url: "https://github.com/ivmai/libatomic_ops/releases/download/v7.6.14/libatomic_ops-7.6.14.tar.gz"
     sha256: "390f244d424714735b7050d056567615b3b8f29008a663c262fb548f1802d292"
+patches:
+  "7.6.14":
+    - patch_file: "patches/enable-cmake-7_6_14.patch"
```

Comment:
please, update the cmake patch fields, https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

The patch_source is essential in this case, it's a big patch and should come from the upstream.

Reply:
Done